test/functional/connection/connection.ts

```diff
@@ -1,10 +1,16 @@
 import "reflect-metadata";
 import {expect} from "chai";
 import {Post} from "./entity/Post";
+import {Guest as GuestV1} from "./entity/v1/Guest";
+import {Comment as CommentV1} from "./entity/v1/Comment";
+import {Guest as GuestV2} from "./entity/v1/Guest";
+import {Comment as CommentV2} from "./entity/v1/Comment";
```

Comment:
I did not understand why are you doing this? GuestV1 and GuestV2 are same just like CommentV1  and CommentV2. What is the point to do it?